project1/src/components/Buttons/FeedTabs.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { withStyles, WithStyles } from "@material-ui/core/styles";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import { styles } from "./styles/FeedTabsStyle";
+
+interface FeedTabsProps {
+  isLoggedIn: boolean;
+  tag: string;
+  isTagToggeld: boolean;
+  feedID: number;
```

Comment:
make a new type called `FeedTab`
```
  selectedFeedTab: FeedTab;
  onChangeSelectedFeedTab: (tab: FeedTab) => void;
```
where
```
type FeedType = "yourFeed" | "globalFeed";
```

this will make the code more readable instead of 0 and 1